cdk/lib/constructs/pollerLambda.ts

```diff
@@ -115,27 +115,34 @@ export class PollerLambda {
 		// allow lambda to write to the ingestion-lambdas queue
 		ingestionLambdaQueue.grantSendMessages(lambda); //TODO consider making that queue a destination for the poller-lambda and then the lambda just returns the payload (on success and failure) rather than using SQS SDK within the lambda
 
+		const { stalledAlarmThreshold, stalledAlarmPeriod } = (() => {
+			if (!pollerConfig.idealFrequencyInSeconds) {
+				// long polling
+				return {
+					stalledAlarmThreshold: 1,
+					stalledAlarmPeriod: Duration.minutes(3), // long polling (where we expect http requests would timeout in much less than 3 mins)
+				};
+			}
+			// fixed frequency polling
+			const period = Duration.minutes(15); // fixed frequency polling (because 15mins is max delay for SQS)
+			const periodInSeconds = period.toSeconds();
+			const threshold = Math.floor((periodInSeconds / pollerConfig.idealFrequencyInSeconds) * 0.8); // * 0.8 to allow for some slack
```

Comment:
v. minor, but variable only used once and method call is as descriptive as the variable
```suggestion
			const threshold = Math.floor((period.toSeconds() / pollerConfig.idealFrequencyInSeconds) * 0.8); // * 0.8 to allow for some slack
```